components/Typography/index.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from 'next/link';
```

Comment:
We need to stop creating files named `index.tsx`. This should be `components/Typography.tsx`.